### PR TITLE
Add initial files

### PR DESCRIPTION
**Pull request by EmreAtes**
These are the initial printers I put in [#428](https://github.com/uuid-rs/uuid/issues/428).

### CODE REVIEW

**Comment by Dylan-DPC-zz** [comment]
I don't know much about python but I hope that's python 3 to begin with 😆

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+## Installation
+Add these lines to the `~/.gdbinit` file:
+```
+python
+import sys
+sys.path.insert(0, '/path/to/uuid-gdb')
+from uuid_printer import register_uuid_printer
+register_uuid_printer()
+end
+```
```
Comment (kinggoesgaming) [inline README.md:13]: maybe split the imports and register steps (`~/.gdbinit` may already have some of the imports)

**Comment by EmreAtes** [comment]
@Dylan-DPC it's python 3, and works for gdb 8.2-7 on fedora, but I don't know if there are older versions of gdb that only support python 2. I can change a couple of things like the class declaration to make it backwards compatible.

File: README.md (hunk 1 above)
Comment (EmreAtes) [reply README.md:13]: I modelled it after this one: https://github.com/ruediger/Boost-Pretty-Printer 

Do you mean a split with a linebreak?

**Pull request merged**